docs/examples/tutorial/parallelization/median.py

```diff
@@ -0,0 +1,38 @@
+# distutils: language = c++
```

Comment:
Are we using C++ language in this file? If I am not wrong not, so we can remove this.

Reply:
Yes. libcpp

Reply:
Yes I see now. I missed that.